src/settings_internal.js

```diff
@@ -269,4 +269,4 @@ var MINIFY_WHITESPACE = true;
 
 var ASYNCIFY_IMPORTS_EXCEPT_JS_LIBS = [];
 
-var WARN_DEPRECATED = true;
+var WARN_DEPRECATED = true;
```

Comment:
Revert this

Reply:
Add the newline back

Reply:
Yes please.  Mostly because we don't want changed to unrelated files as part of a PR.   I could would `git checkout upstream/main <filename>` to revert the file (add `-p` to just revert part of it).